en/02_Developer_Guides/15_Customising_the_Admin_Interface/03_CMS_Layout.md

```diff
@@ -35,17 +34,16 @@ This causes the framework to:
 to the layout manager)
 - trigger `redraw` on children which also cascades deeper into the hierarchy (this is framework activity)
 
-[notice]
-Caveat: `layout` is also triggered when a DOM element is replaced with AJAX in `LeftAndMain::handleAjaxResponse`. In
-this case it is triggered on the parent of the element being replaced.
-Calling the top level `layout` is not enough as it will wrongly descend down the detached element's hierarchy.
-[/notice]
-
-[notice]
-Caveat: invocation order of the `redraws` is crucial here, generally going from innermost to outermost elements.  For
-example, the tab panels have be applied in the CMS form before the form itself is layouted with its sibling panels to
-avoid incorrect dimensions.
-[/notice]
+> [!WARNING]
+> There are some caveats to this:
+>
+> `layout` is also triggered when a DOM element is replaced with AJAX in `LeftAndMain::handleAjaxResponse`. In
+> this case it is triggered on the parent of the element being replaced.
+> Calling the top level `layout` is not enough as it will wrongly descend down the detached element's hierarchy.
+>
+> The invocation order of the `redraws` is crucial here, generally going from innermost to outermost elements.  For
+> example, the tab panels have be applied in the CMS form before the form itself is layouted with its sibling panels to
+> avoid incorrect dimensions.
```

Comment:
Merged these together to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.